apps/freenet-ping/app/src/main.rs

```diff
@@ -0,0 +1,150 @@
+use std::{collections::HashSet, time::Duration};
+
+use clap::Parser;
+use freenet_stdlib::{
+    client_api::{ClientRequest, ContractRequest, ContractResponse, HostResponse, WebApi},
+    prelude::{
+        ContractContainer, Parameters, RelatedContracts, StateDelta, UpdateData,
+        WrappedState,
+    },
+};
+use names::Generator;
+
+#[derive(Debug, Default, serde::Serialize, serde::Deserialize)]
+struct Ping {
+    from: HashSet<String>,
+}
+
+#[derive(clap::Parser)]
+struct Args {
+    #[clap(long, default_value = "localhost:50509")]
+    host: String,
+    #[clap(long, default_value = "info")]
+    log_level: tracing::level_filters::LevelFilter,
+}
+
+#[tokio::main(flavor = "multi_thread")]
+async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    let args = Args::parse();
+    tracing_subscriber::fmt()
+        .with_ansi(true)
+        .with_level(true)
+        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+        .with_max_level(args.log_level)
+        .with_line_number(true)
+        .init();
+
+    const PING_CODE: &[u8] =
+        include_bytes!("../../contracts/ping/build/freenet/freenet_ping_contract");
+
+    // create a websocket connection to host.
+    let uri = format!(
+        "ws://{}/contract/command?encodingProtocol=native",
+        args.host
+    );
+    let (stream, _resp) = tokio_tungstenite::connect_async(&uri).await.map_err(|e| {
+        tracing::error!(err=%e);
+        e
+    })?;
+    let mut client = WebApi::start(stream);
+    // put contract first
+    let params = Parameters::from(vec![]);
```

Comment:
let's add an option in the cmd args to pass a string that we can pass as bytes as a parameter in case we want to put multiple while testing out, and add a default string `freenet-ping` for parameter